Build fails in Docker image

## Dockerfile

```dockerfile
ARG SWIFT_VERSION=latest
FROM swift:${SWIFT_VERSION}

LABEL maintainer "417-72KI <417.72ki@gmail.com>"

ARG MINT_REVISION=master
ENV MINT_REVISION=${MINT_REVISION}

# Install Mint
RUN git clone -b "${MINT_REVISION}" --depth 1 "https://github.com/yonaskolb/Mint.git" ~/Mint && \
    cd ~/Mint && \
    swift run mint install yonaskolb/mint && \
    cd && \
    rm -rf ~/Mint

CMD [ "/bin/bash" ]
```

## Error

```sh
/root/Mint/Sources/MintKit/ProcessInfoExtensions.swift:12:61: error: use of unresolved identifier '_SYS_NAMELEN'
        let data = Data(bytes: &sysinfo.machine, count: Int(_SYS_NAMELEN))
                                                            ^~~~~~~~~~~~
```

It seems to be caused by #185.

--- a/.github/workflows/swift.yml
+++ b/.github/workflows/swift.yml
@@ -5,7 +5,7 @@ on:
   pull_request: {}
 
 jobs:
-  build:
+  macOS:
     runs-on: macOS-latest
     steps:
     - uses: actions/checkout@v2
@@ -15,3 +15,14 @@ jobs:
       run: swift build
     - name: Run tests
       run: swift test 2>&1 | xcpretty
+  linux:
+    runs-on: ubuntu-latest
+    container: swift:5.2
+    steps:
+      - uses: actions/checkout@v2
+      - name: Resolve
+        run: swift package resolve
+      - name: Build
+        run: swift build
+      - name: Run tests
+        run: swift test 2>&1

--- a/Sources/MintKit/ProcessInfoExtensions.swift
+++ b/Sources/MintKit/ProcessInfoExtensions.swift
@@ -1,6 +1,7 @@
 import Foundation
 
 extension ProcessInfo {
+    #if os(macOS)
     /// Returns a `String` representing the machine hardware name or nil if there was an error invoking `uname(_:)` or decoding the response.
     ///
     /// Return value is the equivalent to running `$ uname -m` in shell.
@@ -14,4 +15,5 @@ extension ProcessInfo {
         guard let identifier = String(bytes: data, encoding: .ascii) else { return nil }
         return identifier.trimmingCharacters(in: .controlCharacters)
     }
+    #endif
 }

--- a/Tests/MintTests/ProcessInfoExtensionTests.swift
+++ b/Tests/MintTests/ProcessInfoExtensionTests.swift
@@ -2,6 +2,8 @@
 import XCTest
 
 final class ProcessInfoExtensionTests: XCTestCase {
+    #if os(macOS)
+
     #if arch(x86_64)
     func testMachineHardwareName_Intel() {
         XCTAssertEqual(ProcessInfo.processInfo.machineHardwareName, "x86_64")
@@ -12,5 +14,7 @@ final class ProcessInfoExtensionTests: XCTestCase {
     func testMachineHardwareName_AppleSilicone() {
         XCTAssertEqual(ProcessInfo.processInfo.machineHardwareName, "arm64")
     }
+    #endif
+
     #endif
 }

